src/Pages/ProductDetails/Components/Thumbnail/Thumbnail.js

```diff
@@ -0,0 +1,125 @@
+import React, { Component } from 'react';
+import { thumbnailData } from './ThumbnailData';
+import './Thumbnail.scss';
+
+class Thumbnail extends Component {
+  constructor() {
+    super();
+    this.state = {
+      number: 0,
+    };
+  }
+
+  handleIncrease = () => {
+    this.setState({
+      number: this.state.number + 1,
+    });
+  };
+
+  handleDecrease = () => {
+    this.setState({
+      number: this.state.number - 1,
+    });
+  };
+
+  render() {
+    return (
+      <section className="top-view">
+        <article className="thumbnail-wrapper">
+          <img
+            src="../../../images/상세페이지 썸네일.jpg"
+            alt="대표 이미지"
+            className="top-img"
+          />
+          <div className="thumbnail-info">
+            <dl className="goods-name">
+              <dt>[춰컬릿] 미니미 믹스 2종 (봉지)</dt>
+              <dd> 부담 없이 즐기는 미니 초콜릿</dd>
+            </dl>
+            <div className="goods-price-wrapper">
+              <div className="goods-price">
+                <span className="price"> 10,500</span>
+                <span className="unit"> 원 </span>
+              </div>
+              <div className="goods-discount">
+                <div className="discount-percent">
+                  <span> 웰컴5%</span>
+                </div>
+                <span className="discount-price"> 개당 525원 적립</span>
+              </div>
+            </div>
+            <div className="goods-dl">
+              {thumbnailData.map(ele => {
```

Comment:
```suggestion
              {thumbnailData.map(thumbnail => {
```
el, ele, data 등의 추상적이고 불명확한 이름은 가급적 피해주세요

Reply:
네~

Reply:
@seod0209 수정이 됐나요?